pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = "The Python lib for digital ocean automation"
 authors = ["Christian Eland <eland.christian@gmail.com>"]
 license = "Apache-2.0"
 readme = "README.md"
-documentation = "https://pydoautomator.readthedocs.io"
+documentation = "https://github.com/christian-hawk/pydoautomator/tree/master/docs"
```

Comment:
Hello @ashutosh007865 . Thanks for contributing.

Correct url is: https://christian-hawk.github.io/pydoautomator/